lib/inherit.js

```diff
@@ -84,7 +79,14 @@ function override(base, res, add) {
 
                         return res;
                     };
-                result.__base = baseMethod;
+
+                result.__base = function() {
+                    var baseMethod = base[name]
```

Comment:
Well... in fact I don't like `this.__base` syntax at all:
- it is ambiguous (programmer should know, that it is a some kind of special "dynamic" property)
- it hides `__base` property
- it doesn't work in next tick
- as we can see it doesn't work well with stubs

May be we should start from classical:
```js
<Parent>.prototype.<method>.apply(this, arguments);
```
and think about how we can simplify it?

To have in the end something like:
```js
this.__parent.<method>(arg1, arg2, ...);
```